src/test/js/log-karaoke/noStages.js

```diff
@@ -90,7 +90,7 @@ module.exports = {
             this
                 .elements('css selector', 'div.result-item.success', function (collection2) {
                     const count2 = collection2.value.length;
-                    this.assert.notEqual(count, count2);
```

Comment:
If this line was the problem, it's just the error message that's gibberish. It failed because it was expecting them to be different and they weren't.